frontend/prettier.config.mjs

```diff
@@ -0,0 +1,9 @@
+import vercelPrettierOptions from '@vercel/style-guide/prettier';
+
+/** @type {import('prettier').Config} */
+const config = {
+  ...vercelPrettierOptions,
+  plugins: [...vercelPrettierOptions.plugins, 'prettier-plugin-prisma'],
```

Comment:
Can we have this too? It's nice for tailwind
  "plugins": [
    "prettier-plugin-tailwindcss"
  ],

Reply:
Ya, I added it

Reply:
It turns out that file wasnt actually even doing anything since we're specifying the config in `package.json`. Are you sure we need to specify it still? Here are the docs for the process I used: https://www.npmjs.com/package/@vercel/style-guide

Reply:
It looks like including the vercel style guide in the package.json comes with these defaults, but we should be able to add our own plugins and such by override? in case we want more in the future

https://github.com/vercel/style-guide/blob/canary/prettier/index.js

Reply:
Yeah I'm still not sure of this config. I might not merge this PR. this was more of an experiment branch to try diff configs. I like the linting rules for this branch but the prettier config is pretty confusing